### DISCUSSION TITLE
这个仓库和f2库之间是什么关系？ 还有这个仓库的Server.py启动之后有什么作用？

### DISCUSSION THREAD

**Discussion by Positliver** [Johnserf-Seed/TikTokDownload#776] [Q&A] [answered]
希望有人能解答一下，这几个仓库之间的关系， 看安装教程没有整体宏观的框架概念， 只能依葫芦画瓢，一步一步按照安装提示操作， 但是文档一会儿跳到f2仓库， 完全没有理解之间的关系， 希望能得到回答

**Comment by Johnserf-Seed** [comment, marked as answer]
## 关于本仓库与 `F2` 仓库的关系
本仓库的早期版本在代码结构和功能实现上存在较多混乱，导致维护和扩展难度较大。为了优化架构并提升使用体验，我们决定将核心功能 封装为独立的 `PyPI` 包，并以 `CLI`  工具的形式进行分发，提供更清晰、模块化的使用体验。

> [!TIP]
>`F2` 将作为底层核心工具库持续迭代，而本仓库将会只专注于旧的业务逻辑与应用。

> [!IMPORTANT]
> - `Server.py` 脚本的核心作用是让旧版用户能够在本地继续调用 `xbogus` 算法。
> - 在当前过渡阶段，它起到了 `兼容` 的作用，确保部分旧版功能仍可使用。

## `F2` 的诞生与定位
- 核心定位：`F2` 是一个全新的架构和工具集，旨在提供稳定的底层支持。
- 双重使用方式：既支持 `CLI` 命令行调用，也支持 `开发者接口（API）`，以便在不同场景下灵活集成。
- 桥接过渡：在重构过程中，本仓库删除了旧版代码，暂时通过调用 `F2` 提供的接口来实现相关功能。（其实是不推荐这样调用的，因为 `CLI` 足够优雅)

## 当前状态与未来规划
### 当前阶段
- 开发重心：当前的所有开发工作都集中在 `F2` 仓库与 ([`F2-APP`](https://github.com/Johnserf-Seed/f2-app))仓库，所有新功能、新接口以及新应用将优先在 `F2` 中发布。
- 桥接过渡：本仓库删除了旧版代码，暂时作为调用 `F2` 接口的桥梁，提供基础功能支持。
- 旧版本维护：最后发布的可执行文件版本可在 [v1.4.2.2 Release 页面](https://github.com/Johnserf-Seed/TikTokDownload/releases/tag/v1.4.2.2) 中提供，但已停止维护。

### 未来规划
- 在 `F2` 发布 `正式版` 后，本仓库将进行全面重构。
- 重构后的本仓库将专注于 `Douyin` 和 `TikTok` 相关业务功能更新，并基于 `F2` 提供的底层支持进行开发和优化。

**Reply by Positliver** [reply]
非常感谢耐心解答， 完全解答了我的疑问， 再次感谢提供这么好的工具，期待项目的持续更新
